File: Projet_final/projetFinalSpringBoot/src/main/java/com/exemple/model/Coordonnees.java
package com.exemple.model;

import javax.persistence.Column;
import javax.persistence.Embeddable;

import com.fasterxml.jackson.annotation.JsonView;

@Embeddable
public class Coordonnees {
	@Column(name = "coordonnees_telephone")
	@JsonView(JsonViews.Common.class)
	private String telephone;
	@Column(name = "coordonnees_email")
	@JsonView(JsonViews.Common.class)
	private String email;

	public String getTelephone() {
		return telephone;
	}

	public void setTelephone(String telephone) {
		this.telephone = telephone;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}
}

File: Projet_final/angularBT/src/app/model/coordonnees.ts
export class Coordonnees {
  
    constructor(private _telephone:string='',
  private _email:string=''){
    
  }
  
  public get telephone(): string {
    return this._telephone;
  }
  
  public set telephone(_telephone: string) {
    this._telephone =_telephone;
  }
  
  public get email(): string {
    return this._email;
  }
  
  public set email(_email: string) {
    this._email =_email;
  }

}